packages/core/data-transfer/types/remote/protocol/server/messaging.ts

```diff
@@ -0,0 +1,10 @@
+import type { ServerError } from './error';
+
+export type Message<T = unknown> = {
+  id?: string;
+  data?: T | null;
+  error?: ServerError | null;
+};
+
+// Successful
+export type OK = Message<{ ok: true }>;
```

Comment:
Something like "OkMessage" would be more clear and match the above "TransferMessage"